jm-kedro-project/src/jm_kedro_project/set_env_variables.py

```
@@ -0,0 +1,5 @@
+import os
+
+os.environ["JAVA_HOME"] = r"C:\Program Files\Eclipse Adoptium\jdk-11.0.27.6-hotspot"
+os.environ["HADOOP_HOME"] = r"C:\hadoop"
+os.environ["PATH"] = r"C:\Program Files\Eclipse Adoptium\jdk-11.0.27.6-hotspot\bin;" + os.environ["PATH"]
```

Ruff (0.11.9)

No newline at end of file

Add trailing newline

(W292)